### PR TITLE
Feature: [MIQ-2130] Export CSV with verbose names

### PR DESCRIPTION
**Pull request by subham-sarkar-guavus**
### CATEGORY

Choose one

- [ ] Bug Fix
- [x] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [x] Build / Development Environment
- [ ] Documentation

### SUMMARY
Export CSV feature now has column names aliases support

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/viz.py [hunk 1]
```diff
@@ -518,6 +522,19 @@ def data(self):
 
     def get_csv(self):
         df = self.get_df()
+        if df is None:
```
Comment (arpit-agarwal) [inline superset/viz.py]: can we comment when we will hit this if block?

File: superset/viz.py [hunk 2]
```diff
@@ -518,6 +522,19 @@ def data(self):
...
+            df = pd.DataFrame()
+            include_index = not isinstance(df.index, pd.RangeIndex)
+            return df.to_csv(index=include_index, **config.get('CSV_EXPORT'))
+        parsed_json = json.dumps(self.datasource.data, sort_keys=True)
```
Comment (arpit-agarwal) [inline superset/viz.py]: `self.datasource.data` is this full data to be exported?

File: superset/viz.py (hunk 1 above)
Comment (subham-sarkar-guavus) [reply superset/viz.py]: When we export CSV for filter box (the charts which don't have any query obj). So, this avoids the error when the query's empty.

File: superset/viz.py (hunk 2 above)
Comment (arpit-agarwal) [reply superset/viz.py]: lets see if we can avoid json_dump and use python object to for extracting information.
Also i think verbose_map is available in `self.datasources.verbose_map`

**Pull request merged**